### ISSUE TITLE
Problem with minification of ngUrlBind.js file with grunt v0.4.5

### ISSUE THREAD

**Issue by StanislavValov** [resin-io/ngUrlBind#2]
I found it when i've deployed my app on remote machine and Unknown Provider error occurred.

Possible solution for that is:
`factory('ngUrlBind',['$location', function($location) {`

Passing dependencies, including the dependency names as strings so that Angular will know what to inject after minification

**Comment by alexandrosm** [comment]
Hi there, I've pushed a new version, can you please let me know if version 0.0.3 fixes the issue?

**Comment by StanislavValov** [comment]
Yes, the new version works and fixes the issue, thanks for your instant help.
